bin/oneoff/backfill_data/backfill_csa_section_course_id

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+CDO.log = Logger.new(STDOUT)
+ActiveRecord::Base.record_timestamps = false
+
+options = {actually_update: false}
```

Comment:
nit: You mentioned being able to use this for other pilots - could we pass `pilot_names` in the options to make this more generic/reusable?

Reply:
Good question. I'm inclined to keep it as is so that it requires testing to do this for another pilot instead of going straight on production (in case it would be a huge number of rows updated). Curious if you think that's a valid concern @sureshc